### PR TITLE
Unify yaml extension

### LINKED ISSUE #917: Unify files extension to .yaml or .yml
### What is the expected behavior?

I would like to propose to unify YAML files extension to one of those two: `.yaml` or `.yml`.

I found problematic configurations in the GitHub Action due to the differences with these extensions.

I don't know if there is a more common extension or another: @IceKhan13 , @psschwei , @akihikokuroda

### PR DESCRIPTION
**Pull request by Tansito**
### Summary

Fix #917 , this pull request pretends to unify YAML extensions to `.yaml`.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -161,7 +161,7 @@ services:
     profiles: [ "full" ]
     volumes:
       - /var/log:/var/log
-    command: -config.file=/etc/promtail/config.yml
```
Comment (Tansito) [inline docker-compose.yaml:164]: @akihikokuroda I had doubts about this configuration file. I didn't find it and I don't know if it's correct to change it. I tested everything and seems to be working but I would like to know your opinion.

Comment (psschwei) [reply docker-compose.yaml:164]: I _believe_ that file is in the container (and we don't touch it), so I don't think we want to update that one

Comment (Tansito) [reply docker-compose.yaml]: Solved in [f553791](https://github.com/Qiskit-Extensions/quantum-serverless/pull/918/commits/f5537916109f0e02752817d8909ad120f837fdca)

**Review by psschwei — APPROVED** [review]

**Comment by psschwei** [comment]
@Tansito just need to resolve the conflicts and we can merge this one

**Pull request merged**